bin/jens-gc

```diff
@@ -23,21 +23,22 @@ def parse_cmdline_args():
     """Parses command line parameters."""
     parser = optparse.OptionParser()
     parser.add_option('-c', '--config',
-        help="Configuration file path (defaults to '/etc/jens/main.conf'",
-        default="/etc/jens/main.conf")
+                      help="Configuration file path "
+                           "(defaults to '/etc/jens/main.conf'",
```

Comment:
It didn't know about that :) However, maybe better in a separate MR. Happy to review it :D